### ISSUE TITLE
`description` selector seems wrong

### ISSUE THREAD

**Issue by keysmashes** [lensvol/fl-duly-noted#2]
https://github.com/lensvol/fl-duly-noted/blob/6eac449e89610f676ff6d9751d1c8f7a3c3c5f07/content.js#L173-L173

This selects `div[class='storylet__description-container'] > p`, but (at least for the storylets I currently have access to) the description is directly within the `div`:

```
<div class="storylet__description-container">Rumours race around the Hill...</div>
```

This breaks notes for any such storylets entirely, with `TypeError: description is null` in the console.

aha, the selector seems valid for Fifth City storylets, though:

```
<div class="storylet__description-container"><p><span class="descriptive">Fate is Fallen London's premium currency...</span></p></div>
```

**Comment by lensvol** [comment]
Good catch! It seems that older storylets (e.g. `Working with the Honey-Addled Detective`) indeed have different layouts compared to more recent ones.  Will fix it soon, stay tuned!

**Comment by lensvol** [comment]
Fixed in 8f9548d32d0ff52f91f899c7fd557c7cdb858ea3
